runtime/connectors/SplitFileRecursive_test.go

```diff
@@ -0,0 +1,21 @@
+package connectors
+
```

Comment:
nit: the filename `runtime/connectors/SplitFileRecursive_test.go` is not idiomatic Go – it should have the same name as the file it tests followed by `_test`, e.g. `connectors_test.go`